README.md

```diff
@@ -42,6 +52,11 @@ REST API for managing cache control information for pages within the legacy CMS
 | IS_PUBLISHING                | false                           | Determines if the instance is in publishing or not                                                                 |
 | ZEBEDEE_URL                  | http://localhost:8082           | Zebedee host address and port for authentication                                                                   |
 
+### Secrets are Auto-Deployed
+When the secrets are deployed to Vault, this will automatically cause Nomad to trigger a redeployment of the application to pick up the new secrets. Please note that this functionality does not appear to work with the current nomad/vault versions, but if this is upgraded it may then be functional. 
+
```

Comment:
The current headline and wording still suggests that this functionality is working even though there is a caveating final sentence clarifying that is not the case. I have suggested alternative wording that makes it clear that although the functionality has been added to the service it is not yet functional in the current infrastructure.
```suggestion
### Auto-Deployment of secrets
Functionality has been added to the nomad plan so that when the secrets are deployed to Vault, this will automatically cause Nomad to trigger a redeployment of the application to pick up the new secrets. Please note that this functionality does not appear to work with the current nomad/vault versions, but if these are upgraded it may then become functional. 

```